tests/notify.rs

```diff
@@ -103,6 +148,11 @@ fn validate_watch_dir<F, W>(ctor: F) where
   let mut w = ctor(tx).unwrap();
 
   w.watch(dir.path()).unwrap();
+  // Windows needs some time for thread spinup before we start creating files.
```

Comment:
Wouldn't it be better to have the watch function block until after `ReadDirectoryChangesW` is called?